sphinx_rtd_theme/layout.html

```diff
@@ -220,16 +220,23 @@
     {%- if theme_analytics_id %}
     <!-- Theme Analytics -->
     <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-    ga('create', '{{ theme_analytics_id }}', 'auto');
-    {%- if theme_analytics_anonymize_ip|tobool %}
-    ga('set', 'anonymizeIp', true);
-    {%- endif %}
-    ga('send', 'pageview');
+      (function () {
+        // New Google Site Tag (gtag.js) tagging/analytics framework
+        // https://developers.google.com/gtagjs
+        var script = document.createElement("script");
```

Comment:
Why are you using JS to create the tag? Why not just create the tag yourself in html. This is a lot more complicated than what is recommended on https://developers.google.com/analytics/devguides/collection/upgrade/analyticsjs

Reply:
Fully agree, resulted from copy-paste laziness. Issue has been addressed